browser/js/movieQueue/movieQueue.html

```diff
@@ -0,0 +1,5 @@
+<div class="container">
+	<div class="row">
+		<hf-moviequeue-item movies="user.movieQueue"></hf-moviequeue-item>
```

Comment:
remove movies="user.movieQue" and replace with ng-repeate movie in user.movieQueue